axis_backend/apps/documents/tests/repositories/test_document_repository.py

```
@@ -0,0 +1,443 @@
+        )
+
+        queryset = self.repository.get_latest_versions()
+
+        # Should only include latest versions
+        for doc in queryset:
+            self.assertTrue(doc.is_latest)
+
+    def test_get_expiring_soon(self):
+        """Test getting documents expiring soon."""
+        # Create document expiring in 15 days
+        expiry_date = date.today() + timedelta(days=15)
```

Ruff (0.14.5)

`datetime.date.today()` used

(DTZ011)